frontend/app/module/polling_stations/page/PollingStationListPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { PollingStationType, usePollingStationList } from "@kiesraad/api";
+import { IconChevronRight } from "@kiesraad/icon";
+
+function pollingStationType(pollingStationType: PollingStationType): string {
```

Comment:
Hi, thanks for your review! 

This function already does an exhaustiveness check: because of the return type string it is not allowed to return undefined when you did not handle all cases (`TS2366: Function lacks ending return statement and return type does not include undefined`). I was myself looking at a `default` with something like `never` but that seemed not to be needed in this case.

Would you still prefer separating out a lookup object?

Reply:
> This function already does an exhaustiveness check: because of the return type string it is not allowed to return undefined when you did not handle all cases (`TS2366: Function lacks ending return statement and return type does not include undefined`). I was myself looking at a `default` with something like `never` but that seemed not to be needed in this case.

I didn't know TS would catch this, fair! The error message is a little more cryptic then when (for example) you leave out a variant in the typed object:
```
typescript [2741]: Property 'Bijzonder' is missing in type '{ Mobiel: string; VasteLocatie: string; }' but required in type '{ VasteLocatie: string; Bijzonder: string; Mobiel: string; }'.
```

> Would you still prefer separating out a lookup object?

I was thinking of creating the typed object instead of the function and just perform the lookup instead of doing a function call. To me it's a little more readable, but I guess it's a matter of taste.

Reply:
> I was thinking of creating the typed object instead of the function ...
Ah yes that turns out to be a better solution, thanks! Implemented in https://github.com/kiesraad/abacus/pull/453/commits/b96547431de16c6db2e0051f082c694b5515d2f1

Reply:
> Shouldn't the PollingStationType be english? looking for instance at:

Good suggestion! Updated the backend enum and all that followed in https://github.com/kiesraad/abacus/commit/b96547431de16c6db2e0051f082c694b5515d2f1